Docs/source/usage/parameters.rst

```diff
@@ -343,7 +343,7 @@ Domain Boundary Conditions
     * ``pec``: This option can be used to set a Perfect Electric Conductor at the simulation boundary. Please see the :ref:`PEC theory section <theory-bc-pec>` for more details. Note that PEC boundary is invalid at `r=0` for the RZ solver. Please use ``none`` option. This boundary condition does not work with the spectral solver.
       If an electrostatic field solve is used the boundary potentials can also be set through ``boundary.potential_lo_x/y/z`` and ``boundary.potential_hi_x/y/z`` (default `0`).
 
-    * ``none``: No boundary condition is applied to the fields with the electromagnetic solver. This option must be used for the RZ-solver at `r=0`.
+    * ``none``: This option must be used for the RZ-solver at `r=0`.
```

Comment:
Was the 'none' going to be changed to 'r_origin'? (Though I like 'axis')

Reply:
yes, but should be a separate PR if that works :)

Reply:
Yes, might be worth in one go.

I think during the meeting most liked `r_origin`, although `r_axis` et al are also reasonable choices.